comms/discovery/server/server.go

```diff
@@ -434,23 +438,46 @@ func (s *ChatServer) getChatPermissions(c echo.Context) error {
 		return c.String(400, "wallet not found: "+err.Error())
 	}
 
-	permission, err := queries.GetChatPermissions(db.Conn, ctx, userId)
-	if err != nil {
-		if err == sql.ErrNoRows {
-			permission = schema.All
-		} else {
+	validatedPermissions := make(map[string]*ValidatedPermission)
+
+	query := c.Request().URL.Query()
+	encodedIds := query["users"]
```

Comment:
super small nit, can this be "id" instead to match `getBulkTracks`? https://audiusproject.github.io/api-docs/#get-bulk-tracks